metadata/reportssql/testing_report_functions.sql

```diff
@@ -1710,3 +1783,59 @@ BEGIN
 
 END$$
 DELIMITER ;
+
+
+-- getDaysBetweenHIVPosAndEnrollment
+
+DROP FUNCTION IF EXISTS getDaysBetweenHIVPosAndEnrollment;
+
+DELIMITER $$
+CREATE FUNCTION getDaysBetweenHIVPosAndEnrollment(
+    p_patientId INT(11)) RETURNS INT(11)
+    DETERMINISTIC
+BEGIN
+    DECLARE enrollmentDate DATE;
+    SET enrollmentDate = getPatientDateOfEnrolmentInProgram(p_patientId, "HIV_PROGRAM_KEY");
+    RETURN (DATEDIFF(enrollmentDate, getHIVTestDate(p_patientId, "2000-01-01", enrollmentDate)));
```

Comment:
@lauravignoli `getHIVTestDate` returns the hiv's test date irrespective of the result, for only positive result test date, we can use the function `getDateOfPositiveHIVResult`